src/frontend/komponenter/Fagsak/Vedtak/FeilutbetaltValuta/FeilutbetaltValutaSkjema.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Label } from '@navikt/ds-react';
+import type { ISODateString } from '@navikt/familie-form-elements';
+import { FamilieInput } from '@navikt/familie-form-elements';
+import { FamilieDatovelger } from '@navikt/familie-form-elements';
+import type { ISkjema } from '@navikt/familie-skjema';
+
+import type { IBehandling } from '../../../../typer/behandling';
+import type { IFeilutbetaltValutaSkjemaFelter } from '../../../../typer/eøs-feilutbetalt-valuta';
+import type { FamilieIsoDate } from '../../../../utils/kalender';
+import {
+    erIsoStringGyldig,
+    FamilieIsoTilFørsteDagIMåneden,
+    FamilieIsoTilSisteDagIMåneden,
+    serializeIso8601String,
+    sisteDagIInneværendeMåned,
+} from '../../../../utils/kalender';
+
+interface IFeilutbetaltValutaSkjemaProps {
+    skjema: ISkjema<IFeilutbetaltValutaSkjemaFelter, IBehandling>;
+}
+
+const FlexDatoInputWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+`;
+
+const FlexRowDiv = styled.div`
+    display: flex;
+`;
+
+const StyledFamilieInput = styled(FamilieInput)`
+    width: 7.5rem;
+    .navds-label {
+        width: 18rem;
+    }
+    .navds-text-field__input {
+        width: 11rem;
+    }
+`;
+
+const gjørOmDatoHvisGyldigInput = (
+    dato: string | undefined,
+    omgjøringsfunksjon: (dato: FamilieIsoDate) => FamilieIsoDate
+): string => {
+    if (dato === undefined) return '';
+    if (erIsoStringGyldig(dato)) return omgjøringsfunksjon(dato);
+    else return dato;
+};
+
+const FeilutbetaltValutaSkjema: React.FunctionComponent<IFeilutbetaltValutaSkjemaProps> = ({
+    skjema,
+}) => {
+    return (
```

Comment:
Kan være pilfunksjon og kan vurdere å ta inn fom, tom, feilutbetaltBeløp og visFeilmeldinger i stedet for skjema

Reply:
Bytter til pilfunksjon